OpenRA.Mods.Common/Widgets/Logic/Ingame/ObserverStatsLogic.cs

```diff
@@ -98,7 +102,8 @@ public class ObserverStatsLogic : ChromeLogic
 				createStatsOption("Economy", economyStatsHeaders, () => DisplayStats(EconomyStats)),
 				createStatsOption("Production", productionStatsHeaders, () => DisplayStats(ProductionStats)),
 				createStatsOption("Combat", combatStatsHeaders, () => DisplayStats(CombatStats)),
-				createStatsOption("Earnings (graph)", earnedThisMinuteGraphHeaders, () => EarnedThisMinuteGraph())
+				createStatsOption("Earnings (graph)", earnedThisMinuteGraphHeaders, () => EarnedThisMinuteGraph()),
+				createStatsOption("Army (graph)", armyThisMinuteGraphHeaders, () => ArmyThisMinuteGraph()),
```

Comment:
Please also change the `150` in the `statsDropDown.ShowDropDown` line below to `155` to prevent a few px overflow in the dropdown height.